change/@fluentui-react-native-checkbox-2020-12-21-13-12-54-sammy-CheckmarkHCBug.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Changing hover/focused color for Checkmark",
```

Comment:
nit keep an eye for contextual information that can be added to make intent easier to understand e.g. make hover and focused colors match for checkmark glyph and background

Reply:
Thanks for the advice, will do!